### PR TITLE
Move verification server related monitoring configs to a sub folder.

### PR DESCRIPTION
**Pull request by yegle**
Move verification server related configs (all configs right now) to a sub folder. We will be creating key server related configs in its own sub folder.

Part of #532

### CODE REVIEW

File: terraform/alerting/main.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module "verification-server" {
```
Comment (icco) [inline terraform/alerting/main.tf:1]: Can we import a module into another module?

Comment (yegle) [reply terraform/alerting/main.tf:1]: Running `tf apply` on my test project seem to work as intended.

**Pull request merged**